<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="message">
     <property name="text">
      <string>Message Here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dontAsk">
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Don't ask again</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="continueAdjust">
        <property name="text">
         <string>Continue with &lt;changed&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="continueRetain">
        <property name="text">
         <string>Continue with &lt;original&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>continueAdjust</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continueRetain</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
